.changeset/nine-camels-cover.md

```diff
@@ -0,0 +1,6 @@
+---
+"@aws-amplify/ui-react": patch
+---
+
+fix: Authenticator issue where InitMachine useEffect runs every render, causing `children` of 
+`Authenticator` to be unmounted and remounted on every render.
```

Comment:
```suggestion
fix: Authenticator issue where InitMachine useEffect runs every render, causing `children` of `Authenticator` to be unmounted and remounted on every render.
```

NIT: You shouldn't need to manually add newlines, markdowns will handle it in changelogs.